.ci/e2eTestingMacosDaily.groovy

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'darwin && orka && x86_64' }
+  environment {
+    REPO = 'e2e-testing'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    ELASTIC_APM_ACTIVE = "true"
+    ELASTIC_APM_ENVIRONMENT = "ci"
+    ELASTIC_APM_LOG_FILE = "stderr"
+    ELASTIC_APM_LOG_LEVEL = "debug"
```

Comment:
I assume these env variables are the ones related to the explicit instrumentation of the e2e-testing with the APM, right?

Reply:
Indeed. They are used for the Elastic APM go agent